autoload/vimlparser.vim

```diff
@@ -2215,6 +2227,7 @@ let s:VimLParser.builtin_commands = [
       \ {'name': 'endtry', 'minlen': 4, 'flags': 'TRLBAR|SBOXOK|CMDWIN', 'parser': 'parse_cmd_endtry'},
       \ {'name': 'endwhile', 'minlen': 4, 'flags': 'TRLBAR|SBOXOK|CMDWIN', 'parser': 'parse_cmd_endwhile'},
       \ {'name': 'enew', 'minlen': 3, 'flags': 'BANG|TRLBAR', 'parser': 'parse_cmd_common'},
+      \ {'name': 'eval', 'minlen': 4, 'flags': 'EXTRA|NOTRLCOM|SBOXOK|CMDWIN', 'parser': 'parse_cmd_eval'},
```

Comment:
I've noted that the help mentions that it can be abbreviated ("ev", "eva"), but that should not be used.
It seems to make sense to parse it anway though?  (i.e. minlen=2)

Reply:
oh... vim help should write it as `ev[al]` anyway 🤔 
thanks!

Reply:
That might be on purpose, because it is discouraged.
Might be the job of vint then to warn about using the full name, but not vimlparser's.

Reply:
yeah, thanks!